system/lib/libcxxabi/include/__cxxabi_config.h

```diff
@@ -71,7 +71,8 @@
 #define _LIBCXXABI_NO_CFI
 #endif
 
-#if defined(__arm__)
+// wasm32 follows the arm32 ABI convention of using 32-bit guard.
+#if defined(__arm__) || defined(__wasm32__)
```

Comment:
Nice.. its a shame we probably still need to define this in `tools/system_libs.py` because fastcomp.

I guess you could update the TODO in `tools/system_libs.py` and wrap it in a check for fastomp instead?

Reply:
how can we check for fastcomp in system_libs.py?
Or I guess we could just update the comment so that when we grep for fastcomp later we find it?

Reply:
`Settings.WASM_BACKEND`

Reply:
!